### DISCUSSION TITLE
MinGW AUR

### DISCUSSION THREAD

**Discussion by tallero** [msys2/MINGW-packages#8197] [Show and tell]
Hi,

I wanted to inform you that I registered the domain http://mingwaur.org and had it redirect to this [gitlab](https://gitlab.com/mingw-aur)/[github](https://github.com/mingw-aur) organization.

The aim is to lighten the load on this repository, provide you better PKGBUILDs and host `-git` versions and other amenities.

If you want to contribute in any way to the project that will certainly be appreciated.

Thanks,
tallero

**Comment by Kreijstal** [comment]
domain is down

**Reply by tallero** [reply]
Things have gone a long road since mingwaur.org @Kreijstal.

I have since the last days of last year, after three years of work, bootstrapped the [Ur](https://thegithub.com/themartiancompany/ur), short for simply User Repository, an uncensorable, permissionless, undeletable, cross-platform, source-unified successor of all source repositories used by environments relying on the pacman package manager hosted on Ethereum Virtual Machine compatible blockchain networks, which currently resides on the Gnosis blockchain at the contract address indicated in its source code and which hosts around 100 cross-platform working uncensorable, undeletable package recipes, among which some very popular videogames, meaninglessly published with the purpose of trying to get a larger public attention.

A plethora of new tools had to be written to make this possible, in particular [reallymakepkg](https://github.com/themartiancompany/reallymakepkg), a cross-platform extension of makepkg, [inteppacman](https://github.com/themartiancompany/inteppacman) a pacman extension to manage Android applications, the [Ethereum Virtual Machine File System](https://github.com/themartiancompany/evmfs), an uncensorable undeletable file host and protocol, the EVM toolchain, a set of libraries and commands to integrate blockchains networks communications into native applications written in any language ([evm-contracts-tools](https://github.com/themartiancompany/evm-contracts-tools), [libevm](https://github.com/themartiancompany/libevm), [evm-wallet](https://github.com/themartiancompany/evm-wallet) and others) as well as the decentralized uncensorable Twitter, which I am not allowed to directly post links to here, the untamperable, decentralized [EVM OpenPGP Key Server](https://github.com/themartiancompany/evm-openpgp-keyserver) and [EVM GnuPG](https://github.com/themartiancompany/evm-gnupg), an OpenPGP implementation not relying on emails, just to make possible for developers to be able to communicate in case in coordinated ad-personam attack towards one or a group of developers aimed at censoring them over all the internet, like it has happened to me after my personal and professional life has been destroyed by the same people who had me write all of these things and I've been thrown out from night to day from YouTube, Arch Linux and other online communities after more than a decade of work and without any explanation. I've also witnessed coordinated censorship on the Stack Exchanges and Wikimedia platforms as well.

The Ur is part of two operating system like things I've been working on for around the same amount of time, one extension of the other, called Life and DogeOS.

Some of the Ur is hosted on [The Martian Company](https://github.com/themartiancompany) Github namespace but that could be taken down, same as my email and other online identities at any time, same at is has already happened for my YouTube channel 'truocolo' and my 'tallero' Aur identities.

I am working to provide an already bootstrapped environment which will be available at https://humaninstrumentalityproject.org, but it's no guaranteed I'll manage to, so be sure to download the sources and build it yourself as soon as you can or like download the binary packages for the Life and DogeOS Android Termux environment or the Arch Linux one from the [Fallback User Repository](https://github.com/themartiancompany/fur) to be sure we can keep in contact if the worse happens.
The bulk of all the sources I've mentioned have already been published on the EVMFS so they can't be deleted anymore by anybody and they can be seamlessly recovered at any time even if everything went to go down, still it would be harder to do without having an already bootstrapped environment available.

As far as I can see even darker and grimmer days are about to come, so be very fast adapting.
I'm damn serious, my life has been turned into a living hell and I'm made to wish to die around every couple days or weeks.
Oh, and just because it seems I'm providing you with better tools to freely communicate please don't evangelize any of them as salvific, as they all have known deficiencies and they are just the best I've been able to do to defend myself from those same people who have chosen not to kill me and so have made possible I'm writing this message here to you today.
I doubt those people can be trusted, they're shady people with no morals, have no respect for basic human rights and they lie all the time.

Love and peace,
thank you for your reply, it has been very appreciated.

Rino, Tallero, Dvorak, Truocolo, Ogoorcs, or however you wanna call me.

**Reply by tallero** [reply]
For reference, here is an [example](https://github.com/themartiancompany/evmfs-ur/blob/master/PKGBUILD) of a complete, uncensorable, cross-platform package recipe, as I haven't had no time in the slightest to write almost any documentation.
Support for MinGW environment in reallymakepkg has still not landed, as it's two years I can barely get access to a computer, let alone a windows machine.
It should seamlessly build from the MSYS2 environment though.
